src/views/Order.vue

```diff
@@ -350,6 +360,12 @@ ion-select {
   margin-inline-start: 0;
 }
 
+ion-card-header {
```

Comment:
this is a global change. Need to update all pages and make rule global.

Reply:
Ok Sir, I have made this rule global.